ArticleTemplates/assets/js/shim.js

```diff
@@ -1,92 +1,97 @@
+        'articleTagInserter',
+        'articleOutbrainInserter',
+        'audioBackground',
+        'superAudioSlider',
+        'audioPlay',
+        'audioStop',
+        'commentsFailed',
+        'articleCommentsFailed',
+        'commentsClosed',
+        'commentsOpen',
+        'showMoreTags',
+        'commentsReplyFormatting',
+        'footballGoal',
+        'footballStatus',
+        'footballMatchInfo',
+        'footballMatchInfoFailed',
+        'liveblogDeleteBlock',
+        'liveblogNewBlock',
+        'getMpuPosCallback',
+        'initMpuPoller',
+        'videoPositioning',
+        'getArticleHeight'
+    ];
 
-var nativeFunctionCalls = [
-    'articleCommentsInserter',
-    'articleCardsInserter',
-    'articleTagInserter',
-    'articleOutbrainInserter',
-    'audioBackground',
-    'superAudioSlider',
-    'audioPlay',
-    'audioStop',
-    'commentsFailed',
-    'articleCommentsFailed',
-    'commentsClosed',
-    'commentsOpen',
-    'showMoreTags',
-    'commentsReplyFormatting',
-    'footballGoal',
-    'footballStatus',
-    'footballMatchInfo',
-    'footballMatchInfoFailed',
-    'liveblogDeleteBlock',
-    'liveblogNewBlock',
-    'getMpuPosCallback',
-    'initMpuPoller',
-    'videoPositioning',
-    'getArticleHeight'
-];
+    Array.prototype.forEach.call(nativeFunctionCalls, function (name) {
+        // Create a function to catch early calls
+        window[name] = function () {
+            // Create or get the queue for this function
+            var queue = name + 'Queue';
+            window[queue] = window[queue] || [];
+            // Store arguments for each call so
+            // true function can apply these when ready
+            window[queue].push(arguments);
+        };
+    });
 
-Array.prototype.forEach.call(nativeFunctionCalls, function (name) {
-    // Create a function to catch early calls
-    window[name] = function () {
-        // Create or get the queue for this function
-        var queue = name + 'Queue';
-        window[queue] = window[queue] || [];
-        // Store arguments for each call so
-        // true function can apply these when ready
-        window[queue].push(arguments);
+    window.applyNativeFunctionCall = function (name) {
+        var queue = window[name + 'Queue'];
+        
+        if (queue) {
+            Array.prototype.forEach.call(queue, function (item) {
+                window[name].apply(this, item);
+            });
+        }
     };
-});
 
-var applyNativeFunctionCall = function (name) {
-    var queue = window[name + 'Queue'];
-    if (queue) {
-        Array.prototype.forEach.call(queue, function (item) {
-            window[name].apply(this, item);
-        });
-    }
-};
-
-window.animFrame = window.requestAnimationFrame || function( callback ){
-    window.setTimeout(callback, 1000 / 60);
-};
+    window.animFrame = window.requestAnimationFrame || function( callback ){
+        window.setTimeout(callback, 1000 / 60);
```

Comment:
I know this is existing code, but I am not sure this is a legitimate polyfill for RAF (maybe it's not supposed to be). Seems pointless to set random timeouts as everything will just be called ~16ms after calling this function, so you might as well just call the callback immediately.

To really polyfill it, you would need to track the time and batch together as per https://gist.github.com/paulirish/1579671